src/main.cpp

```diff
@@ -142,25 +148,25 @@ char getLoaderChar(){
  * PM2.5 and PM10 read and visualization
  **/
 void sensorLoop(){
+  Serial.print("-->[HPMA] read..");
   int try_sensor_read = 0;
   String txtMsg = "";
   while (txtMsg.length() < 32 && try_sensor_read++ < SENSOR_RETRY) {
     while (hpmaSerial.available() > 0) {
       char inChar = hpmaSerial.read();
       txtMsg += inChar;
-      Serial.print("-->[HPMA] read "+String(getLoaderChar())+"\r");
```

Comment:
@danielbernalb these a lines was approved in a previous PR, we need discuss it. Please ever review the PRs for not will do unnecessary reverts in the future.

Reply:
Yes, is a problem founded after the last rev, the visualization of the serial frame.
In Visual code looks good but in Realterm and Arduino serial terminal the visualization is very confused.
The debug of the frame is very difficult and I choose to use the old routine for this test.

Reply:
while we find a possible solution on the terminals on windows, I going to revert to original devel change